recipes/museval/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Please submit an issue upstream for this to be in the sdist.

Reply:
You mean in the PyPi packaged source distribution ?

Reply:
yup!